drivers/sensor/max17055/max17055.c

```diff
@@ -72,6 +94,29 @@ static int capacity_to_ma(unsigned int rsense_mohms, int16_t val)
 	return rem;
 }
 
+/**
+ * @brief Convert capacity in milliamphours to MAX17055 units
+ *
+ * @param rsense_mohms Value of Rsense in milliohms
+ * @param val_mha Value in milliamphours to convert
+ * @return corresponding value in MAX17055 units, ready to write to a register
+ */
+static int capacity_to_max17055(unsigned int rsense_mohms, uint16_t val_mha)
+{
+	return (val_mha * rsense_mohms) / 5.0;
+}
+
+/**
+ * @brief Convert voltage in millivolts to MAX17055 units
+ *
+ * @param val_mv Value in millivolts to convert
+ * @return corresponding value in MAX17055 units, ready to write to a register
+ */
+static int voltage_mV_to_max17055(uint16_t val_mv)
+{
+	return (val_mv * 16) / 1.25;
```

Comment:
drop brackets?

Reply:
Similar motivation to above - I'm looking for a large value to divide before rounding.
As above, I'd welcome thoughts on using floating point for these functions?